### ISSUE TITLE
layers-slices misses a default import combined with inline type specifiers

### ISSUE THREAD

**Issue by conarti** [conarti/eslint-plugin-feature-sliced#38]
## Behaviour

`layers-slices` does not report an upward import when a default specifier is combined with inline type specifiers only. The default specifier is a value import and should be reported.

## Reproduction

Default config, file `src/entities/bar/model/index.ts`:

```ts
import config from 'src/features/auth';                   // reported
import * as ns from 'src/features/auth';                  // reported
import config, { FooType } from 'src/features/auth';      // reported, on the source string
import config, { type FooType } from 'src/features/auth'; // not reported
```

Expected for the last line: `can-not-import` on `config`.
Actual: no message.

## Where

- `src/rules/layers-slices/model/specifiers/extract-import-specifiers.ts` keeps only `ImportSpecifier` nodes, so `ImportDefaultSpecifier` and `ImportNamespaceSpecifier` never enter the list.
- `validate-specifiers.ts` filters the inline type specifiers out, which leaves an empty invalid list.
- `has-errors-at-all-specifiers.ts` compares 0 invalid against 1 specifier, returns false, so `validate()` in `model/validate-and-report.ts` returns nothing.

A bare default import still reports because both counts are 0 there. The third line above shows the related asymmetry: with a default plus a value named specifier, every named specifier is invalid, so the report is promoted to the whole declaration and lands on the source string; the default specifier itself is never inspected.

Related: #35, #37 (other gaps found while writing the v2 README).
